### DISCUSSION TITLE
Custom Volume Sliders

### DISCUSSION THREAD

**Discussion by Tim0nster** [punxaphil/custom-sonos-card#173] [Ideas]
The Volume Slider is a Little sensitive, when using Homeassisstant on the Phone.

My Idea would be that the Maximal Volume could be Adjusted per Speaker

or 100% Volume will just show, if the Percentage is above 50% with the Dynamic slider ;)

**Comment by punxaphil** [comment]
I'm not sure I fully understand what you are asking for. Is there something you want to change with the dynamic volume slider feature?
